File: C_C++/10178.cpp
#include <cstdio>

int main(){
  int T; scanf("%d", &T);
  while(T--) {
    int candy, sib; scanf("%d %d", &candy, &sib);
    printf("You get %d piece(s) and your dad gets %d piece(s).\n", candy / sib, candy % sib);
  }
}

File: Python/10178.py
count = 0
test_n = int(input())
while count < test_n :
    S = input().partition(" ")
    candy = int(S[0])
    person = int(S[2])
    print("You get", int( candy / person ),"piece(s) and your dad gets", candy % person,"piece(s).")
